packages/material-ui/src/Tab/Tab.js

```diff
@@ -45,23 +45,21 @@ const TabRoot = experimentalStyled(ButtonBase, {
 })(({ theme, styleProps }) => ({
   /* Styles applied to the root element. */
   ...theme.typography.button,
-  maxWidth: 264,
-  minWidth: 72,
+  maxWidth: 360,
+  minWidth: 90,
   position: 'relative',
   minHeight: 48,
   flexShrink: 0,
-  padding: '6px 12px',
+  padding: '12px 16px',
   overflow: 'hidden',
   whiteSpace: 'normal',
   textAlign: 'center',
-  [theme.breakpoints.up('sm')]: {
-    minWidth: 160,
```

Comment:
This change actually makes the `minWidth` smaller on `up-sm`. From `160px` to `90px`. Please add more detail on the migration page thanks.

Reply:
Rollback to old behavior:

```ts

        MuiTab: {
            styleOverrides: {
                root: {
                    // up-sm
                    '@media screen and (min-width: 600px)': {
                        minWidth: 160,
                    },
                },
            },
        },
```

Reply:
The described outcome is expected. I guess we can clarify https://next.material-ui.com/guides/migration-v4/#tab